app/events/db/db_table_field_mappings.yaml

```
@@ -826,3 +826,83 @@ mappings:
         value_post_processors:
             profiles_json: app.events.db.value_post_processors.cast_to_json
             ignored_profiles_json: app.events.db.value_post_processors.cast_to_json
+
+
+    professional_contents_fetch_data:
+        name: professional_contents_fetch_data
+        schema: iiq_schema
+        unique: professional_contents_fetch_request_iiq_id, platform_content_id
+        fields:
+            id: professional_contents_fetch_request_iiq_id
```

id could lead to confusion that, it is content id, but since it is actually job_id, we can rename it to that
```suggestion
            job_id: professional_contents_fetch_request_iiq_id
```

---

but this `id` is the column name that we are getting from response. Our table column name is `professional_contents_fetch_request_iiq_id` so we cannot change the `id` column

---

Isn't the id being populated here?

https://github.com/insightiq-ai/iiq-managed-service/blob/99e9957b2947ab23bb8101db43cf31f281ee3927/app/events/db/db_event_handler.py#L336-L340